functions/lib/clients/twitterClient.js

```diff
@@ -0,0 +1,69 @@
+const Twitter = require('twitter')
+
+exports.makeTwitterClient = function makeTwitterClient(consumer_key, consumer_secret, access_token, access_token_secret) {
```

Comment:
Yeah, I think this makes sense; there's only one purpose for the twitter client, and that's to post the image to twitter, so I think it's good to have the functionality self contained within a Twitter class. This is done now. 

I used the "_func" style to denote private methods that are used for the chunked media upload flow and for sending the actual tweet. Is there a better way to do this (or alternative style preference)?

Turning this into a class also simplified some things — specifically, we no longer need the wrapper interface. The class instance is still mocked in fakes.js, now with only the `tweetMedia` method. 

I think this was the last remaining item you mentioned. Thanks for the feedback and comments! Let me know if you think there's anything else that ought to be fixed/adjusted.